spec/index.html

```diff
@@ -665,6 +671,10 @@ <h3>IRIs</h3>
           in IRIs [[RFC3492]]</li>
         <li>IRIs that are not in Unicode [=Normalization Form C=] [[I18N-Glossary]]</li>
       </ul>
+      <p class="issue" data-number="63">
+        Change Punnycode-encoding refernece from [[RFC3492]] to [[RFC5890]].
```

Comment:
The changes are  #132 so maybe remove this note.

FWIW It's "punycode".

```suggestion
        Change Punycode-encoding refernece from [[RFC3492]] to [[RFC5890]].
```